js/views/message.js

```diff
@@ -152,7 +217,6 @@ views.NoSearchResultMessageListView = Marionette.ItemView.extend({
 	template: "#no-search-results-message-list-template",
 
 	onRender: function() {
-		$('#load-more-mail-messages').hide();
 	}
```

Comment:
I'm confused. This was deleted a while back and isn't showing up for me. Which commit ID is this?